cdc/processor.go

```diff
@@ -653,6 +653,7 @@ func (p *processor) addTable(ctx context.Context, tableID int64, startTs uint64)
 				if pEvent == nil {
 					continue
 				}
+				log.Info("sorter output", zap.Reflect("pevent", pEvent))
```

Comment:
Is this log necessary?

Reply:
it's a debug code, I forgot to remove it.